common/remote_file.h

```diff
@@ -37,11 +37,13 @@
 #endif  // CENTIPEDE_DISABLE_RIEGELI
 
 #if defined(__APPLE__)
-#if (defined(__MAC_OS_X_VERSION_MIN_REQUIRED) &&       \
-     __MAC_OS_X_VERSION_MIN_REQUIRED < __MAC_10_15) || \
-    (defined(__IPHONE_OS_VERSION_MIN_REQUIRED) &&      \
-     __IPHONE_OS_VERSION_MIN_REQUIRED < __IPHONE_13_0)
-// std::filesystem requires macOS 10.15+ or iOS 13+.
+#if (defined(__MAC_OS_X_VERSION_MIN_REQUIRED) &&          \
+     __MAC_OS_X_VERSION_MIN_REQUIRED < __MAC_10_15) ||    \
+    (defined(__IPHONE_OS_VERSION_MIN_REQUIRED) &&         \
+     __IPHONE_OS_VERSION_MIN_REQUIRED < __IPHONE_13_0) || \
+    (defined(__TV_OS_VERSION_MIN_REQUIRED) &&             \
+     __TV_OS_VERSION_MIN_REQUIRED < __TVOS_13_0)
+// std::filesystem requires macOS 10.15+ or iOS 13+ or tvOS 13+.
```

Comment:
This change enables running a wider range of fuzz tests on tvOS, but it's unrelated to the current PR title/description. You could either send another PR or treat this one as a wider "add support for tvOS" one (I'd go with the former).

Reply:
The comment above hasn't been addressed yet.

Furthermore, this code as-is does not work as intended because `__IPHONE_OS_VERSION_MIN_REQUIRED` is also defined for tvOS (see `AvailabilityInternal.h` in Apple's SDK), and its value is hardcoded to `__IPHONE_9_0`. I think you'll need to include `TargetConditionals.h` and check the target OS in addition to the version numbers.